src/cookbooks/bigquery.md

```diff
@@ -1,386 +1,18 @@
 # Accessing and working with BigQuery
 
-This guide will give you a quick introduction to working with data stored
-in [BigQuery](https://cloud.google.com/bigquery/)
+With the transition to [GCP](https://cloud.google.com) in 2019 having been completed, BigQuery has become Mozilla's primary data warehouse and SQL Query engine.
 
-BigQuery uses a columnar data storage format called [Capacitor](https://cloud.google.com/blog/products/gcp/inside-capacitor-bigquerys-next-generation-columnar-storage-format) which supports semi-structured data.
+The following topics provide an introduction to working with data that is stored
+in [BigQuery](https://cloud.google.com/bigquery/):
 
-There is a cost associated with using BigQuery based on operations. As of right now we pay an on-demand pricing for queries based on how much data a query scans. To minimize costs see [_Query Optimizations_](bigquery.md#query-optimizations). More detailed pricing information can be found [here](https://cloud.google.com/bigquery/pricing).
+- [Access](firefox-data-docs/src/cookbooks/bigquery/access.md)
+- [Querying Tables](firefox-data-docs/src/cookbooks/bigquery/querying.md)
+- [Optimization](firefox-data-docs/src/cookbooks/bigquery/optimization.md
```

Comment:
You need to use relative path syntax here: `.` indicates the current directory (`src/cookbooks/`) so we can use that as an anchor.

```suggestion
- [Accessing BigQuery](./bigquery/access.md)
- [Writing BigQuery Queries](./bigquery/querying.md)
- [Optimizing BigQuery Queries](./bigquery/optimization.md)
```